osbs/build/build_response.py

```
@@ -119,6 +119,19 @@ def get_logs(self, decode_logs=True):
         else:
             return logs
 
+    def get_error_message(self):
+        try:
+            str_metadata = graceful_chain_get(self.get_annotations_or_labels(), "plugins-metadata")
+            metadata_dict = json.loads(str_metadata)
+            error = metadata_dict['errors'].items()[0]
+            if error[1]:
+                # Plugin has non-empty error description
+                return "Error in plugin %s: %s" % (error[0], error[1])
+            else:
+                return "Error in plugin %s" % (error[0])
+        except:
+            return ""
```

What's the point of masking all errors, shouldn't this be done by the caller?

If we decide to keep it I'd suggest at least using `except Exception` so as to not block things like signals.